common/xml/convert_and_merge_composable_tests/attribute-cleanup.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<!-- Clean up attributes
+   * Normalize space around attributes on a tag
+   * Sort the attributes by name
+   * Strip xml:base attribute
```

Comment:
Sorry, what do you mean by xml:base attribute?

Reply:
xsltproc automatically adds a `xml:base` attribute to all the tags referencing which xsd they refer to.